bootstrap-css.md

```diff
@@ -0,0 +1,82 @@
+# Bootstrap
+
+## What is Bootstrap?
+
+Bootstrap is a CSS framework by Twitter. It's a collection of 1 CSS file and 1 JavaScript File. 
+
+You can find the documentation [here](http://getbootstrap.com/css/)
+
+## Why use a framework?
+
+Writing your own CSS is way too much work. Web Development can be difficult to start sometimes, and Bootstrap CSS makes that easier. Why write CSS if you don't need to?
+
+## Features
+
+* Stylized Buttons
+* Responsive Navbar
+* Responsive Scaffolding System
+
```

Comment:
Yeah, just the CDN should be good. You'll have to introduce npm to them before teaching them how to install it locally.